lazy.ts

```diff
@@ -14,18 +14,18 @@ export function lazy<T>(factory: () => Type<T>, name?: string): Type<T> {
 }
 export class IslyLazy<T = unknown> extends Type<T> {
 	readonly class = "lazy"
-	backend: Type<T>
-	constructor(protected readonly factory: () => Type<T>, name?: string) {
-		super(name ?? (() => (this.backend ??= factory()).name), () => (this.backend ??= factory()).condition)
+	#backend: Type<T> | undefined
+	get backend(): Type<T> {
+		return (this.#backend ??= this.factory())
 	}
-	is = (value: T | any): value is T => (this.backend ??= this.factory()).is(value)
-	createFlaw(value: any): Omit<Flaw, "isFlaw" | "type" | "condition"> {
-		return this.createFlawFromType((this.backend ??= this.factory()), value)
+	constructor(protected readonly factory: () => Type<T>, name?: string) {
+		super(name ?? (() => this.backend.name), () => this.backend.condition)
 	}
-	getBackend() {
-		return this.backend
+	is = (value: T | any): value is T => (this.#backend ??= this.factory()).is(value)
+	override createFlaw(value: any): Omit<Flaw, "isFlaw" | "type" | "condition"> {
+		return this.createFlawFromType(this.backend, value)
 	}
-	public get(value: any): T | undefined {
-		return this.backend.get(value)
+	public override get(value: any): T | undefined {
+		return this.backend?.get(value)
```

Comment:
looks like this `?` should not be needed?

Reply:
Correct, was an older work around until the getter was added.